Sources/TSCUtility/InterruptHandler.swift

```diff
@@ -8,6 +8,8 @@
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
 */
 
+#if !canImport(Musl)
+
 import TSCLibc
 import TSCBasic
 
```

Comment:
Declarations in this file are deprecated and we're not going to use them when building for Musl.